system-tests/e2e-suite/src/test/scala/com/hotels/styx/proxy/UnwiseCharactersSpec.scala

```diff
@@ -66,7 +66,7 @@ class UnwiseCharactersSpec extends FunSpec with StyxProxySpec {
 
       decodedRequest(req)
 
-      recordingBackend.verify(receivedRewrittenUrl("/url/unwiseQQblah"))
+      recordingBackend.verify(receivedRewrittenUrl("/url/unwise%51%51blah"))
```

Comment:
The logic here is
* Unwise says Q is not legal, changes Q -> %51
* This is as if the client sent a %51
* Thus, styx should NOT change %51 to a Q, even though it's legal to do so